deploy/olm-catalog/application-monitoring-operator/1.4.0/applicationmonitoring.integreatly.org_applicationmonitorings_crd.yaml

```diff
@@ -0,0 +1,645 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
I could be wrong, but I don't think this CRD is supposed to be generated like this

Reply:
@grdryn thanks for taking a look. I'm not really sure here - looking back through the history of the file, it looks like it it has always been `apiVersion: apiextensions.k8s.io/v1beta1`?

Reply:
@obrienrobert I'm more referring to the file as a whole, rather than this particular line. E.g. compare it with the state of this file in the previous operator bundle, there are many differences, because we generally revert the generated changes to this file: https://github.com/integr8ly/application-monitoring-operator/blob/master/deploy/olm-catalog/application-monitoring-operator/1.3.0/applicationmonitoring.integreatly.org_applicationmonitorings_crd.yaml

I can't remember why we do that, or if it's necessary any longer, just pointing it out :)